profiles/common/modules/custom/nexteuropa_formatters/nexteuropa_formatters_fields/nexteuropa_formatters_fields.module

```diff
@@ -57,6 +57,26 @@ function nexteuropa_formatters_fields_field_formatter_info() {
         'show_translations' => TRUE,
       ),
     ),
+    'ne_social_icon' => array(
+      'label' => t('NE Social Icon'),
+      'description' => t('Displays as a Social Icon'),
+      'field types' => array(
+        'link_field',
+      ),
+      'settings' => array(
+        'type' => 'email',
```

Comment:
I don't understand why you are using an email type?

Reply:
This is the formatter's default config.

Reply:
See: https://api.drupal.org/api/drupal/modules%21field%21field.api.php/function/hook_field_formatter_info/7.x

Reply:
I confirm, it's the default value for this type of component.
I don't want put a social network (like facebook, twitter or other) as a default, so I choose one value that exists and that every body knows. 
We can try to put with no default value.

Reply:
👍